### PR TITLE
Adds commonly used `cargo make` tasks

### PR DESCRIPTION
**Pull request by D4nte**
As per feedback in #613, added more details on how to run tests.
Also explains the labels related to issues for wanna-be contributors

---
- [x] RTM (Ready To Merge)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,13 +30,21 @@ The public key can be found on [coblox.tech](https://coblox.tech/security_coblox
...
 ## Feature Requests
 
 To request a change on the way COMIT works, please head over to the [RFCs repository](https://github.com/comit-network/rfc) and checkout the [RFC contributing guidelines](https://github.com/comit-network/rfc/blob/master/CONTRIBUTING.md).
  
 For changes related to our implementation, please [create a feature request](/issues/new/choose) and follow the GitHub template.
 
+## Fixing an issue
+
+If you wish to pick up an existing issue and resolve it, please be aware of the following labels:
+- ✅ [`groomed`](https://github.com/comit-network/comit-rs/labels/groomed): it has been discussed and should contain enough details to get started
```
Comment (LLFourn) [inline CONTRIBUTING.md]: ❓ Are these emojis helpful? How are the labels associated with them?

Comment (D4nte) [reply CONTRIBUTING.md]: It means "yes, you can pick up issues labelled with X" "no, you should not pick up issues labelled with Y".

Happy to remove if it's confusing.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,13 +30,21 @@ The public key can be found on [coblox.tech](https://coblox.tech/security_coblox
...
+- 🤔 [Unlabelled](https://github.com/comit-network/comit-rs/issues?q=is%3Aopen+no%3Alabel): it has not yet been reviewed for design, please avoid
+- 🛑 [`icebox`](https://github.com/comit-network/comit-rs/labels/icebox): it is not the right time to proceed with this issue, please do not pick it
+- 🛑 [`sprint-backlog`](https://github.com/comit-network/comit-rs/labels/sprint-backlog): it will be picked up in our next sprint, please do not pick it
+- 🛑 [`candidate-contribution`](https://github.com/comit-network/comit-rs/labels/candidate-contribution): it has been specially groomed to be used as part of our hiring process, please do not pick it
```
Comment (luckysori) [inline CONTRIBUTING.md]: :question: Should we include a link to wherever people can apply for a job?

File: CONTRIBUTING.md (hunk 1 above)
Comment (luckysori) [reply CONTRIBUTING.md]: I like the emojis.

File: CONTRIBUTING.md (hunk 2 above)
Comment (D4nte) [reply CONTRIBUTING.md]: I like the idea!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -30,13 +30,21 @@ The public key can be found on [coblox.tech](https://coblox.tech/security_coblox
...
 For changes related to our implementation, please [create a feature request](/issues/new/choose) and follow the GitHub template.
 
+## Fixing an issue
+
+If you wish to pick up an existing issue and resolve it, please be aware of the following labels:
+- [`groomed`](https://github.com/comit-network/comit-rs/labels/groomed): it has been discussed and should contain enough details to get started
+- [Unlabelled](https://github.com/comit-network/comit-rs/issues?q=is%3Aopen+no%3Alabel): it has not yet been reviewed for design, please avoid
+- [`icebox`](https://github.com/comit-network/comit-rs/labels/icebox): it is not the right time to proceed with this issue, please do not pick it
+- [`sprint-backlog`](https://github.com/comit-network/comit-rs/labels/sprint-backlog): it will be picked up in our next sprint, please do not pick it
+- [`candidate-contribution`](https://github.com/comit-network/comit-rs/labels/candidate-contribution): it has been groomed to be used as part of our hiring process, please do not pick it; [keen to join?](https://tenx.workable.com/jobs/689268)
```
Comment (bonomat) [inline CONTRIBUTING.md]: 🤡What do you think of ordering the labels differently? I would propose: 
* Unlabelled
* icebox 
* groomed
* candidate-contribution
* sprint-backlog

i.e. first the labels we don't want someone to take.
Also, should we add: `good first issue` ?

Comment (bonomat) [inline CONTRIBUTING.md]:
```suggestion
- [`candidate-contribution`](https://github.com/comit-network/comit-rs/labels/candidate-contribution): it has been groomed to be used as part of our hiring process [keen to join?](https://tenx.workable.com/jobs/689268)
```
I think we should be happy for any contribution and we should not stop someone from doing tickets :)

Comment (D4nte) [reply CONTRIBUTING.md]: So I actually hesitated to add a line about `good-first-issue` but I thought the label is actually very explicit in itself so no need to over explain.

Comment (D4nte) [reply CONTRIBUTING.md]: Well, actually we want people to only pick up `groomed`, right?

Comment (D4nte) [reply CONTRIBUTING.md]: ok are you thinking the same about `sprint-backlog` then?

**Pull request merged**